lighthouse-core/audits/manifest-display.js

```diff
@@ -53,9 +53,14 @@ class ManifestDisplay extends Audit {
     return ManifestDisplay.generateAuditResult({
       rawValue: hasRecommendedValue,
       displayValue,
-      debugString: 'Manifest display property should be set.'
+      debugString: this.createDebugString(hasRecommendedValue)
```

Comment:
I have somewhat mixed feeling about `debugString: undefined` but can live with what you have.

-

I was thinking we'd remove `createDebugString` and just roll with:

```
const auditResult = ManifestDisplay.generateAuditResult({
  rawValue: hasRecommendedValue,
  displayValue
});
if (!hasRecommendedValue) {
  auditResult.debugString = 'Manifest display property should be set.';
}
return auditResult;

```

That way, the `debugString` property only gets added when there's an error.

`debugString: hasRecValue ? undefined : debugString;` works but it stills add the property to the artifacts object. Oher parts of the codebase may make a check like `'debugString' in auditResult`, which would return `true` even though there was no error.